Misc/NEWS.d/next/Library/2024-12-21-03-20-12.gh-issue-128131.QpPmNt.rst

```diff
@@ -0,0 +1 @@
+Completely support random access of uncompressed unencrypted :class:`!zipfile.ZipExtFile`.
```

Comment:
Maybe we don't need to say that it's only for ZipExtFile but only for ZipFile? I don't think we can construct an explicit instance of that class except via `.open()`. Instead,
let's say something about `zipfile.ZipFile.open` in 'r' mode, e.g.:

```suggestion
Completely support random access of uncompressed unencrypted read-only
zip files obtained by :meth:`ZipFile.open <zipfile.ZipFile.open>`.
```